### DISCUSSION TITLE
Add Favourite from recently played

### DISCUSSION THREAD

**Discussion by THFourteen** [OnionUI/Onion#517] [Ideas]
Hey - an idea for a new feature, once I've played a game, or my son has played a few games, it would be great to go through the recently played list and add a few of those games to the favourites list.

**Comment by Znidz** [comment]
This is a good idea and it's something I've wanted to be able to do as well.

**Comment by Sichroteph** [comment]
Thanks for your suggestion. This is something we wait from miyoo to implement. On this specific part we cant for now improve the user experience.
